alert_group_nl_log2slack.py

```diff
@@ -382,6 +388,10 @@ def fetch_logs_with_retry():
 def fetch_logs_and_publish_forever():
     already_published = set()
 
+    if HEALTH_FILE != '':
+        with open(HEALTH_FILE, 'w') as f:
+            os.utime(HEALTH_FILE)
```

Comment:
Deze os.utime() is een no-op, want de open(..., 'w') van de contextmanager doet dit al impliciet.

Dus, `pass` hier is voldoende.

Reply:
removed `os.utime` and added the script to the Readme